libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/schema/EnumAsClassBuilder.kt

```diff
@@ -86,7 +86,7 @@ internal class EnumAsClassBuilder(
                 .returns(selfClassName)
                 .addCode(
                     CodeBlock.builder()
-                        .beginControlFlow("switch($rawValue)")
+                        .beginControlFlow("switch ($T.requireNonNull($rawValue))", JavaClassNames.Objects)
```

Comment:
TIL about `Objects.requireNonNull`. Should we use that instead of https://github.com/apollographql/apollo-kotlin/blob/39df250747de5e3028dc6c11076a2315984d05d5/libraries/apollo-api-java/src/main/java/com/apollographql/apollo3/api/java/Assertions.java#L4?

Reply:
I was almost sure we had something like that but couldn't find it anymore 😅. Ours is a bit nicer since you can pass a message - so I'd keep it.

Reply:
Can you add a comment on ours? 

```
// A version of Objects.requireNonNull that allows a customized message
```